Logical operators (#181)
javascript-tutorial/it.javascript.info: 1-js/02-first-steps/11-logical-operators/article.md

L'operatore OR `||` si comporta come segue:

- Valuta gli operandi da sinistra a destra.
- Ogni operando viene convertito a booleano. Se il risultato è `true`, il logical OR si ferma e ritorna il valore originale dell'operando.
- Se tutti gli operandi sono stati valutati e nessuno è `true`, ritorna l'ultimo operando.

> **Un valore viene ritornato nella sua forma originale, non nella sua conversione booleana.

Reviewer: ```suggestion
Un valore viene ritornato nella sua forma originale, non nella sua conversione booleana.
```

2. **Valutazione a Corto-Circuito.**

>     Gli operandi, oltre che valori, possono essere anche espressioni arbitrarie. L'operatore OR esegue la valutazione da sinistra a destra e si ferma al primo risultato vero, che viene ritornato. Il processo è chiamato "valutazione a corto-circuito" perchè cerca di concludersi il prima possibile (quando possibile), senza aver raggiunto l’operando successivo. 

Reviewer: ```suggestion
    Gli operandi, oltre che valori, possono essere anche espressioni arbitrarie. L'operatore OR esegue la valutazione da sinistra a destra e si ferma al primo risultato vero, il quale viene ritornato. Il processo è chiamato "valutazione a corto-circuito" perché cerca di concludersi il prima possibile, senza dover elaborare tutti gli operandi. 
```

L'operatore accetta un solo argomento e si comporta come segue:

1. Converte l'operando a booleano: `true/false`.

> 2. Ritorna il valore **inverso.

Reviewer: ```suggestion
2. Ritorna il valore inverso.
```
Anche precedentemente ho notato un "bold" (**) non chiuso.... è voluto o un refuso?

Author: Refuso perche' ho scordato di chiudere. Io **inverso** lo lascerei bold, penso aiuti. O preferisci attenerti alla versione originale? Dovrei evitare di prendermi questa liberta' in prossime review?

Reviewer 2: opinione personale, io toglierei il grassetto.. mi sembra che il paragrafo renda già chiaro di cosa si sta parlando, quindi non è necessario mettere in risalto il fatto che il valore si inverte...
Per quanto riguarda prendersi la libertà di fare modifiche, non vorrei dare regole stringenti, vale sempre il discorso del buon senso... se una frase tradotta letteralmente non ha senso, chiaramente vanno apportate delle modifiche... in questo specifico caso però non mi sembra un valore aggiunto necessario

Author: Got it! :)

alert( !!null ); // false
```

Quello che accade è che il primo NOT converte l'operando a booleano e ritorna il suo inverso, e il secondo NOT lo *inverte nuovamente*. Il risultato è un valore di tipo booleano.

> C'è un modo molto più lungo per fare la stessa cosa -- una funzione *built-in* di `Boolean`:

Reviewer: ```suggestion
C'è un modo molto più lungo per fare la stessa cosa, usare la funzione `Boolean`, integrata in JavaScript:
```
